src/openttd.cpp

```diff
@@ -201,6 +201,7 @@ static void ShowHelp()
 		"  -x                  = Never save configuration changes to disk\n"
 		"  -X                  = Don't use global folders to search for files\n"
 		"  -q savegame         = Write some information about the savegame and exit\n"
+		"  -Q                  = Don't scan for NewGRF files (quick startup)\n"
```

Comment:
I think this will give people the impression using this gives them a better experience (for most mere mortals it will not). Maybe make more clear what the impact is? (Maybe by removing quick startup remark? Dunno .. describing things is hard :D)×

Alternatively, for network stuff we use the env to all developers to modify things normal users shouldn't,  to kinda avoid this problem. Not sure if that is a road to take here, but just mentioning. 

Either way, please also document QQ :)